giraffe/src/utils/getTextMetrics.ts

```diff
@@ -3,6 +3,40 @@ export interface TextMetrics {
   height: number
 }
 
+const addPaddingToSampleText = (text: string): string => {
+  if (typeof text !== 'string') {
+    return text
+  }
+
+  let sample = text
+
+  /* The following are considered skinny characters:
+    -
+    1
+    |
+    i
+    l
+    I
+    !
+  */
+
+  for (let index = 0; index < text.length; index += 1) {
+    const char = text.charAt(index)
+    if (
+      char === '-' ||
+      char === '1' ||
+      char === '|' ||
+      char === 'i' ||
+      char === 'l' ||
+      char === 'I' ||
+      char === '!'
+    ) {
```

Comment:
alternative es6 foxy form:
```ts
const SLENDER_BOIS = ['-', '1', '|', 'i', 'l', 'I', '!']

if (SLENDER_BOIS.includes(char)) {
  sample += char
}
```

Reply:
i like this, will use it.

Reply:
Quick quesiton. How long is this "text" usually?

Reply:
10 million characters. No, just kidding. A typical tick label on an axis. For example, "1.00", "1.10". Or even time, "2020-11-09 1:09:33 PM PST". Generally, not very long because it is visual information.

Reply:
haha. ah i see. thank you!